src/store/project.ts

```diff
@@ -88,11 +88,24 @@ export const projectStore = createPartialStore<ProjectStoreTypes>({
 
         const projectFileErrorMsg = `VOICEVOX Project file "${filePath}" is a invalid file.`;
 
+        let buf: ArrayBuffer;
+        try {
+          buf = await window.electron.readFile({ filePath });
+        } catch (e) {
+          await window.electron.showMessageDialog({
+            type: "error",
+            title: "エラー",
+            message: "プロジェクトファイルの読み込みに失敗しました。",
+          });
+          await context.dispatch("LOG_ERROR", {
+            message: String(e),
+          });
```

Comment:
たぶんこっちのがエラー内容がリッチかもです
```suggestion
          await context.dispatch("LOG_ERROR", {
            error: e,
          });
```